lib/src/support/font.cpp

```diff
@@ -150,18 +150,22 @@ namespace cycfi { namespace elements
       {
          FcConfig*      config = FcInitLoadConfigAndFonts();
 
+         std::vector<fs::path> paths = font_paths;
+
 #ifdef __APPLE__
-         auto resources_path = get_user_fonts_directory();
+         paths.push_back(get_user_fonts_directory());
 #else
-         auto resources_path = (fs::current_path() / "resources").generic_string();
+         if (paths.empty())
+            paths.push_back(fs::current_path() / "resources");
 #if defined(ELEMENTS_HOST_UI_LIBRARY_WIN32)
          TCHAR windir[MAX_PATH];
          GetWindowsDirectory(windir, MAX_PATH);
-         auto fonts_path = (fs::path(windir) / "fonts").generic_string();
-         FcConfigAppFontAddDir(config, (FcChar8 const*)fonts_path.c_str());
+         paths.push_back(fs::path(windir) / "fonts");
 #endif
 #endif
-         FcConfigAppFontAddDir(config, (FcChar8 const*)resources_path.c_str());
+         for (auto& path : paths)
+            FcConfigAppFontAddDir(config, (FcChar8*)path.generic_string().c_str());
```

Comment:
Doesn't want the `const` ? It's probably good practice to use c++ casting style (I know I sometimes forget :-) ).

Reply:
Yes, I would definitely recommend named casts. It's not about shorter code, it's about clear code for the reader. Also, `-Wold-style-cast` and Core Guidelines...